decred_memes

```diff
@@ -0,0 +1,12 @@
+| Title               | decred meme collection |
+| Author              | OfficialCryptos |
+| Featured text       |Decred meme|
+
+Thesis statement for art collection - What the article is about, and what is it aiming to prove
+
+# Article Title
+
+Meme Piece 1
+Meme Piece 2
```

Comment:
![DCR Guy](https://github.com/PhoenixGreen/Cypherpunktimes-articles/assets/158556488/66b7eeaf-be99-430e-8c07-b7a70474bc7a)

Reply:
Love the image but we’re not going to be able to use this one due to copyright issues.